Carbon.IIS/Functions/Add-CIisHttpHeader.ps1

```diff
@@ -0,0 +1,70 @@
+function Add-CIisHttpHeader
+{
+    <#
+    .SYNOPSIS
+    Adds a new header to the IIS configuration.
+
+    .DESCRIPTION
+    The `Add-CIisHttpHeader` function adds a new header to the IIS configuration. If adding a header for a specific

```

Comment:
Make sure to give instructions on how to use the function, not just what it does. Rephrase to

```text
The `Add-CIisHttpHeader` function adds an HTTP header to IIS. Pass the header's name to the `Name` parameter
 and the header's value to the `Value` parameter. By default, the header is added to all HTTP responses. To add 
the header only to responses to a specific website, application, virtual directory, or directory, pass the path to 
that location to the `LocationPath` parameter.

The function adds the HTTP header by adding it to the `system.webServer/httPrototocl/customHeaders` 
configuration collection. 
```